### ISSUE TITLE
Turbine Test should be updating files rather than only calling the update functions

### ISSUE THREAD

**Issue by maysunfaisal** [eclipse/codewind#1080]
**Codewind version:**
**OS:**

**Che version:**
**IDE extension version:**
**IDE version:**
**Kubernetes cluster:**

**Description of the enhancement:**

Turbine functional tests currently only fake file updates and call the functions. I think it would be better if we can actually update files and call the functions

**Proposed solution:**

**Comment by ssh24** [comment]
I don't think that would be functional test because we are not testing the update function directly and beats the purpose of functional test. What you are saying is a valid scenario but should be done by integration tests IMO
